### ISSUE TITLE
Isochrone open space

### ISSUE THREAD

**Issue by svdhoorn** [GIScience/openrouteservice#494]
When I request an isochrone with the following I receive a blank space (not reached by the isochrone) where I would expect it to be reached. See screenshot below, the arrow points at te space I would expect to be filled.

The request:
```
curl -X POST \
  'https://api.openrouteservice.org/v2/isochrones/driving-car?' \
  -H 'Content-Type: application/json' \
  -d '{"locations":[[5.7836,52.256465]],"range":[3000],"range_type":"distance"}'
```

![afbeelding](https://user-images.githubusercontent.com/41985373/55790104-f9edb480-5abb-11e9-9bd3-ef64a72bf898.png)

**Comment by nilsnolde** [comment]
Hm, that does look weird! It's likely a geometry simplification issue, e.g. that area was reached by the algorithm (obviously, as it's just a few meters away), but the geometry generation somehow simplifies it entirely wrong. We'll look into it soon. Isochrones have similar issues lately, see #468 for example.

**Comment by svdhoorn** [comment]
Thanks for the quick response

**Comment by rabidllama** [comment]
in this particular case it seems to be due to a simplification stage before the generation of the polygons. Basically, when points are added that represent the roads, if they are too close together then only one is added to the list. It should be the case that we can reduce the size of this check so that points have to be closer together before they are skipped, but this should only be done for smaller areas else it could massively increase the complexity of the following generalisation stage, especially when there are very large isochrones being generated

**Comment by svdhoorn** [comment]
Thanks for the explanation; this makes sense

**Comment by rabidllama** [comment]
Fixed in Release 5.0.2
